tests/it/writer.rs

```diff
@@ -28,6 +28,16 @@ fn test_trace_printing() {
 "#]]
     );
 
+    let s = write_traces_with_creation_codes(&tracer);
+    let raw = r#"
+  [209257] → new <unknown>@0xBd770416a3345F91E4B34576cb804a576fa48EB1(<BYTECODE>)
+    └─ ← [Return] 1045 bytes of code
```

Comment:
should the deployed code be displayed too?

Reply:
That's a good question. I need this feature because there's just no way to obtain the creation code, but the deployed code can be easily obtained after the transaction is finished unless there's some selfdestruction happening. I personally won't need the deployed code, but if you see value in printing it, I can add it too.

Reply:
I think that it would actually be very neat even if I don't need it. I think that I should rename `write_creation_codes` to `with_bytecodes` and make that flag trigger printing both creation and deployed codes to avoid having too many knobs.

Reply:
Pushed in the newest commit.